relay-server/src/actors/events.rs

```diff
@@ -1594,7 +1593,12 @@ impl Handler<HandleEnvelope> for EventManager {
                 }
 
                 match processed.envelope {
-                    Some(envelope) => Ok(envelope),
+                    Some(envelope) => {
+                        // TODO: Fix scope problem and uncomment
+                        // envelope_summary.replace(EnvelopeSummary::compute(&envelope));
```

Comment:
@jan-auer Can you advise on this? I'm not sure whether I need to propagate `envelope_summary` through all the `and_then` closures up to this point, enclose them all in one big parent closure (which is what makes this simpler in `common.rs`, AFAICT), or something else.